DEVELOPMENT.md

```diff
@@ -2,39 +2,45 @@
 
 This document gives an overview of the code contained in this monorepo and the recommended development setup.
 
-## Quick start
-
-Get up and running with the recommended development setup. It's advised to develop in the provided container since we depend on some system dependencies (e.g. GDAL).
-
-1. Make a copy of the file ["template.env"](https://github.com/ResonantGeoData/RD-WATCH/blob/phase-ii/template.env) as ".env" and fill it out
-2. Pull in the Docker images: `docker compose --profile vscode pull`
-3. Install the dependencies for Django: `docker compose run --rm poetry install --all-extras`
-4. Install the dependencies for Vue: `docker compose run --rm npm install`
-5. Run migrations: `docker compose run --rm poetry run django-admin migrate`
-6. Load test data: `docker compose run --rm poetry run django-admin loaddata testdata`
```

Comment:
I think it may be better to have a [Django management command](https://docs.djangoproject.com/en/4.2/howto/custom-management-commands/) that creates sample data using the ORM. This would eliminate the need to update the sample data whenever a new migration is introduced.

Reply:
Agreed that would be much better.  Although we want some reasonable locations/polys and states tested so it would need to be curated sample data.